
<!-- saved from url=(0039)http://users.cecs.anu.edu.au/~u1024708/ -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=windows-1252">
<style type="text/css">
.style3 {
	font-size: x-large;
}
.style4 {
	font-size: large;
}
.style5 {
	font-weight: normal;
}
.style6 {
	font-weight: normal;
	font-size: large;
}
</style>


</head><body data-new-gr-c-s-check-loaded="14.1087.0" data-gr-ext-installed=""><table style="width: 100%;">
	<tbody><tr>

		<td style="width: 300px;" class="style3"><strong>
		<img alt="Yu Lin" src="img/people/yu.jpg" height="282"></strong></td> <td style="width: 682px;"><span class="style3"><strong>Yu Lin </strong></span><br> <br>

		<span class="style4">Senior Lecturer</span><br class="style4">
		<span class="style4"><a href="https://cs.anu.edu.au/">School of Computing</a></span><br class="style4">
                <span class="style4"><a href="https://cecs.anu.edu.au/">College of Engineering and Computer Science</a></span><br class="style4">
		<span class="style4"><a href="http://www.anu.edu.au/">Australian National University</a></span><br class="style4">
		<br>
		<span class="style4"><strong>E-Mail: </strong>yu.lin AT anu.edu.au</span><br class="style4">
		<br>
		<span class="style4"><strong>Address: </strong>R4,25, HN Building 145, Acton ACT 2601 </span><br class="style4">
		<br>
                <span class="style4"><strong>Group Website: </strong><span class="style4"><a href="https://cgg-anu.github.io/">https://cgg-anu.github.io</a></span>  </span><br class="style4">
                <br>
                </td>

		
	</tr>
</tbody></table>
<div style="height: 18px;"></div>
<hr size="1">
<div style="height: 18px;"></div>


<span class="style4">I am currently a Senior Lecturer at the <a href="https://cs.anu.edu.au/">School of Computing</a>, <a href="https://www.anu.edu.au/">the Australian National University</a>. Prior to this, I was a postdoctoral scholar, hosted by <a href="http://cseweb.ucsd.edu/~ppevzner/">Prof. Pavel A. Pevzner</a>, at <a href="https://www.cs.ucsd.edu/">the Department of Computer Science and
Engineering</a>, <a href="https://ucsd.edu/">the University of California, San Diego</a>, and I received my Ph.D. in computer science from <a href="http://www.epfl.ch/">EPFL</a>, and my advisor was <a href="http://people.epfl.ch/bernard.moret">Prof. Bernard M.E. Moret</a>.
<div style="height: 8px;"></div>
<br>

<strong><span class="style3">Education<br></span></strong>
</span><span class="style4">
<div style="height: 5px;"></div>
<li>Ph.D. in Computer Science, <a href="http://www.epfl.ch/">EPFL</a>, Lausanne, Switzerland.</li>
<div style="height: 3px;"></div>
<li>M.S. in Computer Science,<a href="http://www.ict.ac.cn/"> Institute of Computing
Technology, Chinese Academy of Sciences</a>, Beijing, China.</li>
<div style="height: 3px;"></div>
<li>B.S. in Computer Science,<a href="http://www.ustc.edu.cn/"> University of Science and Technology of China</a>, Hefei, China.</li>
<div style="height: 15px;"></div>
</span>



<strong><span class="style3">Research Interests<br></span></strong>
<span class="style4">
<div style="height: 5px;"></div>
<li>Genome Assembly and Metagenomics</li>
<div style="height: 3px;"></div>
<li>Comparative Genomics and Phylogeny Reconstruction</li>
<div style="height: 3px;"></div>
<li>Computational Proteomics and Analytical Chemistry</li>
<div style="height: 3px;"></div>
<li>Network Analysis and Data Management</li>
<div style="height: 15px;"></div>
</span>

<strong><span class="style3"><a href="https://cgg-anu.github.io/people.html">PhD Students</a><br></span></strong>
<span class="style4">
<div style="height: 5px;"></div>
<li>Yanbo Li, ANU School of Computng (primary supervisor, 08/2017-02/2022)</li>
<div style="height: 3px;"></div>
<li>Anuradha Wickramarachchi, ANU School of Computing (primary supervisor, 08/2018-02/2022)</li>
<div style="height: 3px;"></div>
<li>Vijini Mallawaarachchi, ANU School of Computing (primary supervisor, 08/2018-02/2022)</li>
<div style="height: 3px;"></div>
<li>Hansheng Xue, ANU School of Computing (primary supervisor, 09/2019-)</li>
<div style="height: 3px;"></div>
<li>Jin Zhao, School of Computer Science, Shandong University (visiting PhD student, 09/2019-09/2020)</li>
<li>Yujia Zhang, ANU School of Computing (primary supervisor, 06/2021-)</li>
<div style="height: 3px;"></div>
<div style="height: 15px;"></div>
</span>

<strong><span class="style3">Publications<br></span></strong>

For a complete list of my publications, please see my <a href="https://cgg-anu.github.io/publicationyu.html">publications</a>, 
<a href="https://scholar.google.com/citations?user=KDwSQ0gAAAAJ">(or Google Scholar page)</a>. My <a href="https://en.wikipedia.org/wiki/Erd%C5%91s_number">Erd√∂s number</a> is 2.
<div style="height: 15px;"></div>


<strong><span class="style3">Program Committee<br></span></strong> 
<span class="style4">
<li> <a href="https://recombcg2021.wordpress.com/">RECOMB-CG (Annual RECOMB Comparative Genomics Satellite Conference) 2021</a></li>
<li> <a href="https://acm-bcb.org/WABI/2021/">WABI (Workshop on Algorithms in Bioinformatics)  2021</a></li>
<li> <a href="https://recombcg2020.wordpress.com/">RECOMB-CG (Annual RECOMB Comparative Genomics Satellite Conference) 2020</a></li>
<li> <a href="http://www.cs.cityu.edu.hk/~shuaicli/apbc2017/">APBC (Asia-Pacific Bioinformatics Conference) 2020</a></li>
<li> <a href="https://recombcg2019.bitbucket.io/">RECOMB-CG (Annual RECOMB Comparative Genomics Satellite Conference) 2019</a></li>
<li> <a href="https://acm-bcb.org/WABI/2019/">WABI (Workshop on Algorithms in Bioinformatics)  2019</a></li>
<li> <a href="http://glab.hzau.edu.cn/APBC2019/">APBC (Asia-Pacific Bioinformatics Conference) 2019</a></li>
<li> <a href="http://algo2018.hiit.fi/wabi/">WABI (Workshop on Algorithms in Bioinformatics)  2018</a></li>
<li> <a href="http://acm-bcb.org/2018/">  ACM BCB (ACM Conference on Bioinformatics, Computational Biology, and Health Informatics) 2018</a></li>
<li> <a href="https://apbc2018.bio.keio.ac.jp/">APBC (Asia-Pacific Bioinformatics Conference) 2018</a></li>
<li> <a href="http://www.acm-bcb.org/WABI/2017/">WABI (Workshop on Algorithms in Bioinformatics)  2017</a></li>
<li> <a href="http://www.cs.cityu.edu.hk/~shuaicli/apbc2017/">APBC (Asia-Pacific Bioinformatics Conference) 2017</a></li>
<li> <a href="https://www.iscb.org/ismb2016">ISMB (International Conference on Intelligent Systems for Molecular Biology) 2016</a></li>
<li> <a href="http://www.ebi.ac.uk/eccb/2016/">ECCB (European Conference on Computational Biology) 2016</a></li>
<li> <a href="http://www.sfasa.org/apbc2016/apbc2016.html">APBC (Asia-Pacific Bioinformatics Conference) 2016</a></li>
<li> <a href="https://www.iscb.org/ismbeccb2015">ISMB/ECCB (International Conference on Intelligent Systems for Molecular Biology &amp; European Conference on Computational Biology) 2015</a></li>
<li> <a href="https://www.iscb.org/ismb2014">ISMB (International Conference on Intelligent Systems for Molecular Biology) 2014</a></li>
</span>
<div style="height: 15px;"></div>

<strong><span class="style3"><a href="https://cgg-anu.github.io/software.html">Software</a><br></span></strong> 
<span class="style4">
<li>GraphBin: A binning tool for metagenomic contigs using assembly graphs. Source code is at <a href="https://github.com/Vini2/GraphBin">Github</a>. </li>
<li>GraphBin2: A new generation of GraphBin that supports overlapped binning of metagenomic contigs. Source code is at <a href="https://github.com/Vini2/GraphBin2">Github</a>. </li>
<li>MetaBCC-LR: Metagenomics binning by coverage and composition for long reads. Source code is at <a href="https://github.com/anuradhawick/MetaBCC-LR">Github</a>. </li>
<li>PlasLR: Adaptation of contig based plasmid classification for error prone long reads to improved genome assembly. Source code is at <a href="https://github.com/anuradhawick/PlasLR">Github</a>. </li>
<li>MultiTrans: Reconstruct transcripts through a mixed integer linear programming on assembly or splicing graphs. Source code is at <a href="https://github.com/jzbio/MultiTrans">Github</a>. </li>
<li>Kmer2SNP: A reference-free and alignment-free variants calling tool for next-generation sequencing data. Source code is at <a href="https://github.com/yanboANU/Kmer2SNP">Github</a>. </li>
<li>DCHap: A divide-and-conquer haplotype phasing algorithm for third-generation sequencing data. Source code is at <a href="https://github.com/yanboANU/Haplotype-phasing">Github</a>. </li>
<li>BacteriaMS-mixture: Bacterial mixture identification using MALDI-TOF mass spectrometry. Source code is at <a href="https://github.com/lmsac/BacteriaMS-mixture">Github</a>. </li>
<li>BaterialMS: Bacterial whole cell typing by mass spectra with bootstrapping assessment. Source code is at <a href="https://github.com/lmsac/BacteriaMS">Github</a>. </li>
<li>FLYE: A de novo assembler of long error-prone reads using repeat graphs. Source code is at <a href="https://github.com/fenderglass/Flye">Github</a>. </li>
<li>ABruijn: A de novo assembler for PacBio and Oxford Nanopore Technologies reads. Source code is at <a href="https://github.com/fenderglass/ABruijn">Github</a>. </li>
<li>MLGO: A web server designed for genome rearrangement and gene order analysis. The MLGO web server is <a href="http://www.geneorder.org/server.php">available</a>.</li>
<li>Matching distance: A distance measure for phylogenetic trees based on matching. The source code and instrustion are <a href="http://users.cecs.anu.edu.au/~u1024708/index_files/matching_distance.zip">downloadable</a>.</li>
<li>TIBA: a tool for phylogeny inference from rearrangement data with bootstrap analysis. The source code and instrustion are <a href="http://users.cecs.anu.edu.au/~u1024708/index_files/TIBA.zip">downloadable</a>.</li>
</span>

</body><grammarly-desktop-integration data-grammarly-shadow-root="true"></grammarly-desktop-integration></html>