
<!-- saved from url=(0055)http://users.cecs.anu.edu.au/~u1024708/publication.html -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=windows-1252">
<style type="text/css">
.style3 {
	font-size: x-large;
}
.style4 {
	font-size: large;
}
.style5 {
	font-weight: normal;
}
.style6 {
	font-weight: normal;
	font-size: large;
}
</style>

</head><body data-new-gr-c-s-check-loaded="14.1087.0" data-gr-ext-installed=""><strong><span class="style3">Publications<br></span></strong>
<a href="https://scholar.google.com/citations?user=KDwSQ0gAAAAJ&amp;hl=en">(Google Scholar)</a>
<ul>
<font size="3" face="Times New Roman">

<li>Wang, Y., Wang, Q., Koehler, H. and Lin, Y., "<a href="https://arxiv.org/abs/2104.09733">
Query-by-Sketch: Scaling Shortest Path Graph Queries on Very Large Networks", 
</a> in press, <em> The ACM Special Interest Group on Management of Data  <a href="https://2021.sigmod.org/">SIGMOD'21</a></em> (2021).	
</li>

<li>Xue, H., Yang, L., Rajan, V., Jiang, W., Wei, Y. and Lin, Y., "<a href="https://arxiv.org/abs/2102.06371">
Multiplex Bipartite Network Embedding using Dual Hypergraph Convolutional Networks",
</a> in press, <em> The Web Conference 2021 <a href="https://www2021.thewebconf.org/">WWW'21</a></em> (2021).
</li>

<li>Wickramarachchi, A., Lin, Y.,
"<a href="https://github.com/anuradhawick/LRBinner">
LRBinner: Binning Long Reads in Metagenomics Datasets",
</a> accepted,  <em>Proc. 21th Workshop on Algorithms in Bioinformatics
</em><em><a href="https://acm-bcb.org/WABI/2021/"> WABI'21</a></em>, (2021).
</li>

<li> Mallawaarachchi, V., Wickramarachchi, A., Lin, Y.,
"<a href="https://almob.biomedcentral.com/articles/10.1186/s13015-021-00185-6">Improving metagenomic binning results with overlapped bins using assembly graphs</a>,"
  <em><a href="http://www.almob.org/">BMC Algorithms for Molecular Biology</a></em> 16:3, (2021).
</li>


<li>Wickramarachchi, A., Lin, Y.,
"<a href="https://github.com/anuradhawick/GraphPlas">
GraphPlas: Refined Classification of Plasmid Sequences using Assembly Graphs", 
</a> <em> The 19th Asia Pacific Bioinformatics Conference <a href="http://www.binfo.ncku.edu.tw/APBC2021/">APBC'21</a></em>, in press at <em> <a href="https://ieeexplore.ieee.org/xpl/RecentIssue.jsp?punumber=8857">IEEE/ACM Trans. on Computational Biology and Bioinformatics</a></em>  (2021).
</li>

<li>Zhao, J., Feng, H., Zhu, D., Lin, Y., 
"<a href="https://github.com/jzbio/MultiTrans">
MultiTrans: a novel algorithm for transcriptome assembly through mixed integer linear programming", 
</a> <em> The 19th Asia Pacific Bioinformatics Conference <a href="http://www.binfo.ncku.edu.tw/APBC2021/">APBC'21</a></em>, in press at <em> <a href="https://ieeexplore.ieee.org/xpl/RecentIssue.jsp?punumber=8857">IEEE/ACM Trans. on Computational Biology and Bioinformatics</a></em> (2021).
</li>

<li>Li, Y., Patel, H., Lin, Y., 
"<a href="https://www.biorxiv.org/content/10.1101/2020.05.17.100305v1">
Kmer2SNP: reference-free SNP calling from raw reads based on matching", 
</a> <em> IEEE International Conference on Bioinformatics and Biomedicine <a href="https://ieeebibm.org/BIBM2020/">BIBM'20</a></em>, pp 208-212,  (2020).
</li>


<li>
  Wickramarachchi, A., Mallawaarachchi, V., Rajan V., Lin, Y.,
  "<a href="https://cgg-anu.github.io/files/MetaBCC-LR.pdf"> MetaBCC-LR: Metagenomics Binning by Coverage and Composition for Long Reads</a>",
  <em>Proc. 28th Conf. on Intelligent Systems for Molecular Biology
  <a href="http://www.iscb.org/ismb2020/">ISMB'20</a></em>, in
  <a href="http://bioinformatics.oxfordjournals.org/"><em>Bioinformatics</em></a> (2020).
</li>

<li>Mallawaarachchi, V., Wickramarachchi, A., Lin, Y., 
"<a href="https://github.com/Vini2/GraphBin2">
GraphBin2: refined and overlapped binning of metagenomic contigs using assembly graphs", 
</a> <em>Proc. 20th Workshop on Algorithms in Bioinformatics
</em><em><a href="http://algo2020.di.unipi.it/WABI2020/"> WABI'20</a></em>, (2020). 
</li>

<li>Mallawaarachchi, V., Wickramarachchi, A., Lin Y., 
"<a href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btaa180/5804980?redirectedFrom=fulltext">
GraphBin: refined binning of metagenomic contigs using assembly graphs", 
</a> <em> <a href="http://bioinformatics.oxfordjournals.org/">Bioinformatics</a></em>, (2020). 
</li>

<li>Li, Y., Lin, Y., 
"<a href="https://ieeexplore.ieee.org/abstract/document/9128055">
DCHap: A divide-and-conquer haplotype phasing algorithm for third-generation sequencing data", 
</a> <em> The 18th Asia Pacific Bioinformatics Conference <a href="http://epigenomics.snu.ac.kr/APBC2020/">APBC'20</a></em>, in press, <em> <a href="https://ieeexplore.ieee.org/xpl/RecentIssue.jsp?punumber=8857">IEEE/ACM Trans. on Computational Biology and Bioinformatics</a></em> (2020).
</li>

<li>Xue, H., Yang, L., Jiang, W., Wei, Y., Hu, Y., Lin, Y., 
"<a href="https://arxiv.org/pdf/2004.01024.pdf">
Modeling Dynamic Heterogeneous Network for Link Prediction using Hierarchical Attention with Temporal RNN", 
</a> <em> The European Conference on Machine Learning and Principles and Practice of Knowledge Discovery in Databases <a href="https://ecmlpkdd2020.net/">ECML-PKDD'20</a></em> (2020).
</li>

<li>Liany, H., Lin, Y., Jeyasekharan, A., Rajan, V., 
"<a href="https://www.biorxiv.org/content/10.1101/2020.02.20.957241v1">
Mining Pathway Associations from Networks of Mutual Exclusivity Interactions", 
</a> <em> 24th International Conference on Research in Computational Molecular Biology, Satellite: Computational Cancer Biology <a href="https://www.recomb2020.org/recomb-ccb">RECOMB-CCB'20</a></em> (2020).
</li>

<li>Iftikhar, M., Wang, Q., Lin, Y., 
"<a href="https://link.springer.com/chapter/10.1007/978-3-030-47436-2_15">
dK-Microaggregation: Anonymizing Graphs with Differential Privacy Guarantees", 
</a> <em> Pacific-Asia Conference on Knowledge Discovery and Data Mining <a href="https://pakdd2020.org/">PAKDD'20</a></em> (2020).
</li>

<li>Yang, Y., Liu, X., Shen, Y., Lin, Y., Yang, P., Qiao, L.,
"<a href="https://www.nature.com/articles/s41467-019-13866-z">
In silico spectral libraries by deep learning facilitate data-independent acquisition proteomics", 
</a> <em> <a href="https://www.nature.com/ncomms/">Nature Communications</a></em>, 11 (1), 1-11, (2020). 
</li>

<li>Kolmogorov, M., Yuan, J., Lin, Y., Pevzner, P., 
"<a href="https://www.nature.com/articles/s41587-019-0072-8">
Assembly of Long Error-Prone Reads Using Repeat Graphs", 
</a> <em> <a href="https://www.nature.com/nbt/">Nature Biotechnology</a></em>, 37, 540-546, (2019). 
</li>
<li>Zhang, Y., Liu, W., Lin, Y., Ng, Y. and Li, S. 
"<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-5470-2">
Large-scale 3D chromatin reconstruction from chromosomal contacts", 
</a> <em> <a href="http://www.biomedcentral.com/bmcgenomics/">BMC Genomics</a></em>, 20 (Suppl 2) :186 (2019)
</li>
<li>Shao, J., Wang, Q., and Lin, Y., 
"<a href="https://www.sciencedirect.com/science/article/pii/S0306437918304770">
Skyblocking for entity resolution", 
</a> <em> <a href="https://onlinelibrary.wiley.com/journal/13652575">Information Systems</a></em>, 85, pp30-43
(2019).
</li>
<li>Iftikhar, M., Wang, Q., Lin, Y., 
"<a href="https://openproceedings.org/2019/conf/edbt/EDBT19_paper_281.pdf">
Publishing Differentially Private Datasets via Stable Microaggregation", 
</a> <em> Proc. 22nd Int'l Conference on Extending Database Technology, <a href="http://edbticdt2019.inesc-id.pt/">EDBT'19</a></em> (2019).
</li>
<li>Farhan, M., Wang, Q., Lin, Y., Mckay, B.
"<a href="https://openproceedings.org/2019/conf/edbt/EDBT19_paper_88.pdf">
A Highly Scalable Labelling Approach for Exact Distance Queries in Complex Networks", 
</a> <em> Proc. 22nd Int'l Conference on Extending Database Technology, <a href="http://edbticdt2019.inesc-id.pt/">EDBT'19</a></em> (2019).
</li>
<li>Wong T., Ranjard, L., Lin, Y., Rodrigo, A.,
"<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2424-7"> 
HaploJuice: accurate haplotype assembly from a pool of sequences with known relative concentrations", 
</a> <em> <a href="http://www.biomedcentral.com/bmcbioinformatics/">BMC Bioinformatics</a></em>, 22;19(1):389 (2018)
</li>
<li>Yang, Y., Lin, Y., and Qiao, L.
"<a href="https://pubs.acs.org/doi/10.1021/acs.analchem.8b02258">  
Direct MALDI-TOF MS Identification of Bacterial Mixtures",</a> 
<em> <a href="https://pubs.acs.org/journal/ancham">Analytical Chemistry,</a></em> 90 (17), pp 10400-0408 (2018)
</li>
<li>Pu, L., Lin, Y., and Pevzner, P.A.
"<a href="https://pubs.acs.org/doi/10.1021/acs.analchem.8b02258"> 
Detection and analysis of ancient segmental duplications in mammalian genomes", 
</a> <em> <a href="https://genome.cshlp.org/">Genome Research</a></em>, 28: 901-909 (2018).
</li>
<li>Xia, R., Lin, Y.,, Zhou, J., Feng, B. and Tang, J., 
"<a href="https://ieeexplore.ieee.org/abstract/document/8345665">
Phylogenetic Reconstruction for Copy-Number Evolution Problems", 
</a> <em> <a href="https://ieeexplore.ieee.org/xpl/RecentIssue.jsp?punumber=8857">IEEE/ACM Trans. on Computational Biology and Bioinformatics</a></em>, (2018).
</li>
<li> Kolmogorov, M., Yuan, J., Lin, Y., and Pevzner, P.A., 
"<a href="https://www.biorxiv.org/content/10.1101/247148v1">
Assembly of Long Error-Prone Reads and Repeat Classification",</a> 
<em>Proc. 22nd Int'l Conf. on Research in Comput. Molecular Bio. <a href="https://recomb2018.fr/">RECOMB'18</a></em> (2018).
</li>
<li> Xia, R., Lin, Y., Zhou, J., Feng, B., and Tang, J., 
"<a href="https://www.liebertpub.com/doi/full/10.1089/cmb.2017.0157"> A Median Solver and Phylogenetic Inference
Based on Double-Cut-and-Join Sorting,"</a> 
<em> <a href="http://www.liebertonline.com/loi/cmb">Journal of Computational Biology </a></em> 25, 3 (2018)
</li>
<li>Feng, B., Lin, Y., Zhou, L., Guo, Y., Friedman, R., Xia, R., Hu, F., Liu, C., and Tang, J.,
"<a href="https://www.nature.com/articles/s41598-017-15484-5">
Reconstructing Yeasts Phylogenies and Ancestors from Whole Genome Data",
</a> <em> <a href="https://www.nature.com/srep/">Scientific Reports</a></em>
7 (1), 15209 (2017)
</li>
<li> Pu, L., Lin, Y., Zhu, D. and Jiang, H. 
"<a href="https://www.sciencedirect.com/science/article/pii/S0304397517306813">Can a breakpoint graph be decomposed into none other than 2-cycles?", </a> 
<em> <a href="https://www.journals.elsevier.com/theoretical-computer-science">Theoretical Computer Science</a></em> (2017)
</li>
<li> Zhou, L., Lin, Y., Feng, B., Zhao, J., and Tang, J., 
"<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-4129-0">Phylogeny analysis from gene-order data
with massive duplications,"
</a> <em> <a href="http://www.biomedcentral.com/bmcgenomics/">BMC Genomics</a></em> 18 (7), 13 (2017)
</li>
<li> Yang, Y., Lin, Y., Chen, Z., Gong, T., Yang, P., Girault, H., Liu, B., and Qiao, L. 
"<a href="https://pubs.acs.org/doi/10.1021/acs.analchem.7b03820"> Bacterial Whole Cell Typing by Mass Spectra Pattern Matching with Bootstrapping Assessment,"</a> 
<em> <a href="https://pubs.acs.org/journal/ancham">Analytical Chemistry </a></em> 89 (22), 12556-12561 (2017).
</li>
<li>
Lin, Y., Yuan, J., Shen, M.W., Kolmogorova, M., Chaissonb, M. and Pevzner, P.A.,
"<a href="http://www.pnas.org/content/early/2016/12/08/1604560113">Assembly of long error-prone reads using de Bruijn graphs,"</a> 
<em> <a href="http://www.pnas.org/">Proc Natl Acad Sci </a></em> 113 (52) E8396-E8405 (2016).
</li>
<li>Zhou, L., Lin, Y., Feng, B., Zhao, J., and Tang, J., 
"<a href="http://www.bookdepository.com/Bioinformatics-Research-Applications/9783319387819">Phylogeny Reconstruction from Whole-Genome Data Using Variable Length Binary Encoding"</a>, 
<em> Proc. 12th Int'l Symp.Bioinformatics Research &amp; Appls. <a href="http://alan.cs.gsu.edu/isbra16/">ISBRA16</a></em>, 345-346, (2016) 
</li>
<li>
Zhou, J., Lin, Y., Rajan, V., Hoskins, W., Feng, B., and Tang, J., 
"<a href="https://almob.biomedcentral.com/articles/10.1186/s13015-016-0088-2">Analysis of gene copy number changes in tumor phylogenetics"</a>, 
 <em> <a href="https://almob.biomedcentral.com/">Algorithms for Molecular Biology</a></em> 11(1) 26, (2016) 
</li><li>
Lin, Y., Shen, M.W., Yuan, J., and Pevzner, P.A., 
"<a href="http://biorxiv.org/content/early/2016/04/13/048413">Assembly of long error-prone reads using de Bruijn graphs",</a> 
<em> Proc. 20th Int'l Conf. on Research in Comput. Molecular Bio. <a href="http://recomb2016.bioinformatics.ucla.edu/"> RECOMB'16</a></em> (2016).
</li>
<li>
Nair, N.U., Hunter, L., Shao, M., Grnarova, P., Lin, Y., Bucher, P., and Moret, B.M.E.,
"<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-2297-3">
  A maximum-likelihood approach for building cell-type trees by lifting",</a>
  <em>Proc. 14th Asia-Pacific Bioinf. Conf. 
  <a href="http://www.sfasa.org/apbc2016/apbc2016.html">APBC'16</a></em>,
<font color="darkviolet"><em>Best Paper Award</em></font>, in <a href="http://www.biomedcentral.com/bmcgenomics"><em>BMC Genomics</em></a>, 17, 1, 14 (2016).
  </li>
<li>
Zhou, J., Lin, Y., Rajan, V., Hoskins, W., and Tang. J., 
"<a href="http://link.springer.com/chapter/10.1007/978-3-662-48221-6_8"> 
Maximum parsimony analysis of gene copy number changes in tumor phylogenetics," </a>
<em>Proc. 15th Workshop on Algorithms in Bioinformatics
</em><em><a href="http://wabi.gatech.edu/"> WABI'15</a></em>,  
<a href="http://www.springer.com/us/book/9783662482209"><em>Lecture Notes in Computer Science</em>
  <strong>9289</strong></a>, 108-120, Springer Berlin Heidelberg, (2015).
  </li>
<li>
Zhou, J., Lin, Y., Hoskins, W., and Tang. J., 
"<a href="http://link.springer.com/chapter/10.1007%2F978-3-319-19048-8_34">
An iterative approach for phylogenetic analysis of tumor progression using FISH copy number,"</a>
 <em>Int'l Symp.Bioinformatics Research &amp; Appls.  <a href="http://grid.cs.gsu.edu/isbra15/"> ISBRA'15</a></em>, 
<a href="http://link.springer.com/book/10.1007/978-3-319-19048-8">
<em>Lecture Notes in Computer Science</em>
  <strong>9096</strong></a>, 402-412. Springer International Publishing (2015).
  </li>
<li>
  Shao, M., Lin, Y., and Moret, B.M.E.,
  "<a href="http://online.liebertpub.com/doi/abs/10.1089/cmb.2014.0096">
  An exact algorithm to compute the DCJ distance for genomes with
  duplicate genes,"</a>
  <a href="http://www.liebertonline.com/loi/cmb">
<em>J. Computational Biology</em></a>
  <strong>22</strong>, 5 (2015), 425-435.
  </li>
<li>
Lin, Y. and Pevzner, P.A., 
<a href="http://link.springer.com/chapter/10.1007%2F978-3-662-44753-6_22"> 
"Manifold de Bruijn Graphs,"</a>
<em>Proc. 14th Workshop on Algorithms in Bioinformatics
<a href="http://algo2014.ii.uni.wroc.pl/wabi/"> WABI'14</a></em>,  
<a href="http://link.springer.com/book/10.1007/978-3-662-44753-6"><em>Lecture Notes in Computer Science</em>
  <strong>8701</strong></a>, 296-310, Springer Berlin Heidelberg, (2014).
  </li>
<li>
Lin, Y., Nurk, S., and Pevzner, P.A., 
"<a href="http://www.biomedcentral.com/1471-2164/15/S6/S6">
What is the difference between the breakpoint graph and the de Bruijn graph?"</a>
  <a href="http://www.biomedcentral.com/bmcgenomics/">
<em>BMC Genomics</em></a> <strong>15</strong>(Suppl 6), S6, (2014).
</li>
<li>
Hu, F., Lin, Y., and Tang, J., 
"<a href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-014-0354-6%22">
 MLGO: phylogeny reconstruction and ancestral inference from gene-order data,
</a>"
  <a href="http://www.biomedcentral.com/bmcbioinformatics/"><em>BMC
  Bioinformatics</em></a>, <strong>15</strong>:354, (2014).  
</li>
<li>
  Nair, N.U., Lin, Y., Manasovska, A., Antic, J., Grnarova, P., Sahu, A.D.,
  Bucher, P., and Moret, B.M.E.,
  "<a href="http://www.biomedcentral.com/1471-2105/15/269">Study of
  cell differentiation by phylogenetic analysis using histone modification
  data</a>,"
  <a href="http://www.biomedcentral.com/bmcbioinformatics/"><em>BMC
  Bioinformatics</em></a>, 15:269 (2014).
  </li>
<li>
  Shao, M., and Lin, Y., and Moret, B.M.E.,
  "<a href="http://link.springer.com/chapter/10.1007/978-3-319-05269-4_22">An exact algorithm to compute the DCJ distance for genomes with duplicate genes</a>,"
  <em>Proc. 18th Int'l Conf. on Research in Comput. Molecular
  Bio. <a href="http://www.compbio.cmu.edu/recomb/">RECOMB'14</a></em>,
  in <a href="http://www.springer.com/computer/bioinformatics/book/978-3-319-05268-7">Lecture Notes in Computer Science 8394</a>, 280-292, Springer Verlag (2014).
  </li>
<li>
  Moret, B.M.E., Lin, Y., and Tang, J.,
  "<a href="http://link.springer.com/chapter/10.1007/978-1-4471-5298-9_7">Rearrangements in phylogenetic inference: Compare, model, or encode?</a>",
  in <a href="http://www.springer.com/computer/bioinformatics/book/978-1-4471-5297-2"><em>Models and Algorithms for Genome Evolution</em></a>,
  vol. 19 of <em>Computational Biology</em> series,
  Chauve, C., et al., eds, Springer Verlag (2013), 147-172.
</li>
<li>
  Lin, Y., Hu, F., Tang, J., and Moret, B.M.E.,
  "<a href="http://psb.stanford.edu/psb-online/proceedings/psb13/lin.pdf">
  Maximum likelihood phylogenetic reconstruction from high-resolution whole-genome data and a tree of 68 eukaryotes</a>,"
  <em>Proc. 18th Pacific Symp. on Biocomputing <a href="http://psb.stanford.edu/">PSB'13</a></em>, pp. 285-296, 2013,  
</li>
<li>
  Nair, N.U., Lin, Y., Bucher, P., and Moret, B.M.E.,
  "<a href="http://link.springer.com/chapter/10.1007/978-3-642-40453-5_25">Phylogenetic analysis of cell types using histone modifications</a>,"
  <em>Proc. 13th Workshop on Algorithms in Bioinformatics <a href="http://algo2013.inria.fr/wabi.shtml">WABI'13</a></em>, in 
  <a href="http://www.springer.com/computer/bioinformatics/book/978-3-642-40452-8">Lecture Notes in Computer Science 8126</a>,
  326-337, Springer Verlag (2013).
</li><li>
  Shao, M., and Lin, Y., and Moret, B.M.E.,
  "<a href="http://www.biomedcentral.com/1471-2105/14/S15/S9">Sorting genomes with rearrangements and segmental duplications through trajectory graphs</a>,"
  <em>Proc. 11th RECOMB Workshop on Comparative Genomics
  <a href="http://rcg2013.sciencesconf.org/">RECOMB-CG'13</a></em>,
  in <em>BMC Bioinformatics</em> 14, Suppl. 15, S9 (2013).
</li>
<li>
  Lin, Y., Rajan, V., and Moret, B.M.E.,
  "<a href="http://www.computer.org/portal/web/csdl/doi/10.1109/TCBB.2011.157">A
  metric for phylogenetic trees based on matching</a>,"
  <a href="http://www2.computer.org/portal/web/tcbb"><em>IEEE/ACM Trans.
  on Computational Biology and Bioinformatics</em></a>,
  9, 4 (2012), 1014-1022.
</li>
<li>
  Lin, Y., Rajan, V., and Moret, B.M.E.,
  "<a href="http://bioinformatics.oxfordjournals.org/content/28/24/3324">TIBA: A tool for phylogeny inference from rearrangement data with bootstrap analysis</a>,"
  <a href="http://bioinformatics.oxfordjournals.org/">Bioinformatics</a> 28, 24 (2012), 3324-3325.
</li>
<li>
Lin, Y., Rajan, V., and Moret, B.M.E.,
  "<a href="http://www.almob.org/content/7/1/21/abstract">Bootstrapping phylogenies inferred from rearrangement data</a>,"
  <em><a href="http://www.almob.org/">BMC Algorithms for Molecular Biology</a></em> <strong>7</strong>:21, 2012.
  <font color="darkviolet"><em>(selected papers from <a href="http://pbil.univ-lyon1.fr/members/sagot/htdocs/wabi2011/wabi2011.html">WABI'11</a>)</em></font>
</li>
<li>
  Shao, M., and Lin, Y.,
  "<a href="http://www.biomedcentral.com/1471-2105/13/S19/S13">
  Approximating the edit distance for genomes under DCJ, insertion
  and deletion</a>,"
  <em>Proc. 10th RECOMB Workshop on Comparative Genomics
  <a href="http://www.uff.br/recombcg/">RECOMB-CG'12</a></em>,
  in <em>BMC Bioinformatics</em> <bf>13</bf> (Suppl 19):S13, 2012.
</li><li>
  Lin, Y., Rajan, V., and Moret, B.M.E.,
  "<a href="http://www.springerlink.com/content/44x101p8p4505115/">Bootstrapping phylogenies inferred from rearrangement data</a>,"
  <em>Proc. 11th Workshop on Algorithms in Bioinformatics <a href="http://pbil.univ-lyon1.fr/members/sagot/htdocs/wabi2011/wabi2011.html">WABI'11</a></em>, 
  <a href="http://www.springer.com/computer/bioinformatics/"><em>Lecture Notes in Computer Science</em> 6833</a>, 175-187, Springer Verlag (2011).
</li><li>
  Lin, Y., and Moret, B.M.E.,
  "<a href="http://www.liebertonline.com/doi/abs/10.1089/cmb.2011.0098">A new genomic evolutionary model for rearrangements, duplications, and
   losses that applies across eukaryotes and prokaryotes</a>,"
  <a href="http://www.liebertonline.com/loi/cmb">
<em>J. Computational Biology</em></a> 18(9): 1055-1064 (2011).
  <font color="darkviolet"><em>(special issue on <a href="http://recombcg.uottawa.ca/recombcg2010/">RECOMB-CG'10</a>)</em></font>.
</li><li>
  Lin, Y., Rajan, V., and Moret, B.M.E.,
  "<a href="http://www.liebertonline.com/doi/abs/10.1089/cmb.2011.0114">Fast and accurate phylogenetic reconstruction from high-resolution
  whole-genome data and a novel robustness estimator</a>,"
  <a href="http://www.liebertonline.com/loi/cmb"><em>J. Computational Biology</em></a> 18(9): 1131-1139 (2011).
  <font color="darkviolet"><em>(special issue on <a href="http://recombcg.uottawa.ca/recombcg2010/">RECOMB-CG'10</a>)</em></font>.

</li><li>
  Lin, Y., Rajan, V., and Moret, B.M.E.,
  "<a href="http://www.springerlink.com/content/978-3-642-21259-8#section=889845&amp;page=1&amp;locus=0">A metric for phylogenetic trees based on matching</a>,"
  <em>Proc. 7th Int'l Symp. Bioinformatics Research &amp; Appls.
  <a href="http://www.cs.gsu.edu/isbra11/">ISBRA'11</a></em>,
  <a href="http://www.springerlink.com/content/978-3-642-21259-8#section=889845&amp;page=1&amp;locus=0"><em>Lecture Notes in Computer Science</em> 6674</a>,
  197-208, Springer Verlag (2011).

</li><li>
  Lin, Y., and Moret, B.M.E.,
  "<a href="http://www.springerlink.com/content/978-3-642-16180-3#section=794587&amp;page=1&amp;locus=0">A new genomic evolutionary model for rearrangements,
  duplications, and losses that applies across eukaryotes and prokaryotes</a>,"
  <em>Proc. 8th RECOMB Workshop on Comparative Genomics
  <a href="http://recombcg.uottawa.ca/recombcg2010/">RECOMB-CG'10</a></em>,
  in
  <a href="http://www.springer.com/computer/bioinformatics/book/978-3-642-16180-3"><em>Lecture Notes in Computer Science</em> 6398</a>, 228-239, Springer Verlag (2010).

</li><li>
  Lin, Y., Rajan, V., and Moret, B.M.E.,
  "<a href="http://www.springerlink.com/content/978-3-642-16180-3#section=794580&amp;page=1&amp;locus=0">Fast and accurate phylogenetic reconstruction from
  high-resolution whole-genome data and a novel robustness estimator</a>,"
  <em>Proc. 8th RECOMB Workshop on Comparative Genomics
  <a href="http://recombcg.uottawa.ca/recombcg2010/">RECOMB-CG'10</a></em>,
  in
  <a href="http://www.springer.com/computer/bioinformatics/book/978-3-642-16180-3"><em>Lecture Notes in Computer Science</em>
  6398</a>, 137-148, Springer Verlag (2010).

</li><li>
  Lin, Y., Rajan, V., Swenson, K.M., and Moret, B.M.E.,
  "<a href="http://www.biomedcentral.com/1471-2105/11/S1/S54">Estimating true
  evolutionary distances under rearrangements, duplications, and losses</a>,"
  <em>Proc. 8th Asia-Pacific Bioinformatics Conf.
  <a href="http://cs.nyu.edu/parida/APBC2010/index.html">APBC'10</a></em>,
  in <a href="http://www.biomedcentral.com/bmcbioinformatics"><em>BMC
  Bioinformatics</em></a> 2010, 11 (Suppl. 1):S54.

</li><li>
  Rajan, V., Xu, A.W., Lin, Y., Swenson, K.M., and Moret, B.M.E.,
  "<a href="http://www.biomedcentral.com/1471-2105/11/S1/S30">Heuristics for the inversion median problem</a>,"
  <em>Proc. 8th Asia-Pacific Bioinformatics Conf.
  <a href="http://cs.nyu.edu/parida/APBC2010/index.html">APBC'10</a></em>,
  in <a href="http://www.biomedcentral.com/bmcbioinformatics"><em>BMC
  Bioinformatics</em></a> 2010, 11 (Suppl. 1):S30.

</li><li>
  Swenson, K.M., Rajan, V., Lin, Y., and Moret, B.M.E.,
  <a href="http://www.liebertonline.com/doi/abs/10.1089/cmb.2009.0184">"Sorting
  signed permutations by inversions in O(nlogn) time,"</a>

  <a href="http://www.liebertonline.com/loi/cmb/"><em>J. Computational Biology</em></a>, 17(3): 489-501 (2010)
  <font color="darkviolet"><em>(special issue on RECOMB'09)</em></font>.

</li><li>
  Swenson, K.M., Lin, Y., Rajan, V., and Moret, B.M.E.,
  "<a href="http://www.liebertonline.com/doi/abs/10.1089/cmb.2009.0156">Hurdles and sorting by inversions: Combinatorial, statistical, and experimental results</a>,"
  <a href="http://www.liebertonline.com/loi/cmb"><em>J. Computational Biology</em></a>, 16(10):1339-1351 (2009)
  <font color="darkviolet"><em>(special issue on RECOMB-CG'08)</em></font>.

</li><li>
  Swenson, K.M., Rajan, V., Lin, Y., and Moret, B.M.E.,
  "<a href="http://www.springerlink.com/content/p6g46228506m/#section=71555&amp;page=1&amp;locus=0">Sorting signed permutations by inversions in O(nlogn) time</a>,"
  <em>Proc. 13th Int'l Conf. on Research in Comput. Molecular Biol.
  <a href="http://www.bio5.org/recomb2009/">RECOMB'09</a></em>, in
  <a href="http://www.springer.com/computer/computational+biology+and+bioinformatics/book/978-3-642-02007-0"><em>Lecture Notes in Computer Science</em>
  5541</a>, 386-399, Springer Verlag (2009).

</li><li>
  Lin, Y., and Moret, B.M.E.,
  "<a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/btn148?ijkey=voEjbY0KCXZLMLU&amp;keytype=ref">Estimating true evolutionary distances under the DCJ model</a>",
  <em>Proc. 16th Conf. on Intelligent Systems for Molecular Biology
  <a href="http://www.iscb.org/ismb2008/">ISMB'08</a></em>, in
  <a href="http://bioinformatics.oxfordjournals.org/"><em>Bioinformatics</em></a> 24(13):i114-i122 (2008).

</li><li>
  Lin, Y., Qiao, Y., Sun, S., Yu, C, Dong G., and Bu, D. "<a href="http://www.springerlink.com/content/wl38115k55105k35/">A fragmentation event model for peptide identification by mass spectrometry,</a>"  <em>Proc. 12th Int'l Conf. on Research in Comput. Molecular Biol. <a href="http://www.comp.nus.edu.sg/~recomb08/">RECOMB'08</a></em>, in <a href="http://www.springerlink.com/content/978-3-540-78838-6#section=223765&amp;page=1"><em>Lecture Notes in Computer Science</em> 4955</a>, 154-166, Springer Verlag (2008).

</li><li>
  Swenson, K.M., Lin, Y., Rajan, V., and Moret, B.M.E.,
  "<a href="http://www.springerlink.com/content/41x15n4k26256753/">Hurdles hardly have to be heeded,</a>"
  <em>Proc. 6th RECOMB Workshop on Comparative Genomics
  <a href="http://igm.univ-mlv.fr/RCG08/">RECOMB-CG'08</a></em>, in 
  <a href="http://www.springer.com/computer/computational+biology+and+bioinformatics/book/978-3-540-87988-6"><em>Lecture Notes in Computer Science</em>5267</a>, 239-249, Springer Verlag (2008).

</li><li>
Wang, L., Lin, Y., and Liu, X., "<a href="http://epubs.siam.org/sicomp/resource/1/smjcat/v38/i4/p1504_s1">Approximation algorithms for Biclustering problems,</a>" <a href="http://epubs.siam.org/sicomp/"><em>SIAM J. Comput.</em></a> 38(4): 1504-1518 (2008).

</li><li>
Sun, S., Yu, C., Qiao, Y., Lin, Y., Dong, G., Liu, C., Zhang, J., Zhang, Z., Cai, J., Zhang, H., and Bu, D.,
"<a href="http://pubs.acs.org/doi/abs/10.1021/pr070479v">Deriving the probabilities of water loss and ammonia loss for amino acids from tandem mass spectra,</a>" <a href="http://pubs.acs.org/journal/jprobs"><em>J. Proteome Res.</em></a>, 7(01): 202-208 (2008).

</li><li>
Yu, C., Lin, Y., Sun, S., Cai, J., Zhang, J., Bu, D., Zhang, Z., and Chen, R., "<a href="http://www.worldscinet.com/jbcb/05/0502a/S0219720007002643.html">An iterative algorithm to quantify factors influencing peptide fragmentation during tandem mass spectrometry,</a>"<a href="http://www.worldscinet.com/jbcb/"><em>J Bioinform Comput Biol.</em></a>, 5(2a):297-311 (2007)  <font color="darkviolet"><em>(special issue on CSB'06)</em></font>.

</li><li>
Wang, L., Lin, Y., andn Liu, X., "<a href="http://www.springerlink.com/content/q068166804303481/">Approximation algorithms for Bi-clustering problems,</a>" <em>Proc. 6th Workshop on Algorithms in Bioinformatics <a href="http://algo06.inf.ethz.ch/wabi">WABI'06</a></em>, in <a href="http://www.springerlink.com/content/978-3-540-39583-6/#section=477143&amp;page=1"><em>Lecture Notes in Computer Science</em> 4175</a>, 310-320, Springer Verlag (2006). 

</li><li>
Yu, C., Lin, Y., Sun, S., Cai, J., Zhang, J., Bu, D., Zhang, Z., and Chen, R., "<a href="http://www.lifesciencessociety.org/CSB2006/toc/353.2006.html">An iterative algorithm to quantify the factors influencing peptide fragmentation for MS/MS spectrum</a>", <a href="http://www.lifesciencessociety.org/CSB2006/"><em>Proc. 2006 LSS Computational Systems Bioinformatics Conference CSB'06</em></a>, 353-360, Imperial College Press (2006)

</li></font></ul><font size="3" face="Times New Roman">
</font><font size="3" face="Times New Roman">
<div style="height: 10px;"></div>



</font>
</body><grammarly-desktop-integration data-grammarly-shadow-root="true"></grammarly-desktop-integration></html>